packages/integrations/node/src/get-network-address.ts

```diff
@@ -0,0 +1,48 @@
+import os from 'os'
+interface ReturnValOpt {
+	local: string[]
+	network: string[]
+}
+
+const wildcardHosts = new Set([
+  '0.0.0.0',
+  '::',
+  '0000:0000:0000:0000:0000:0000:0000:0000',
+])
+type Protocol = 'http' | 'https'
+
+export function getNetworkAddress(protocol: Protocol = 'http', hostname: string, port:number, base?:string) {
```

Comment:
Should the `hostname` be `hostname: string | undefined` since you're handling `undefined` in the code too?

---

Also it would be good to comment where this function is referenced from (Vite)